akash.html

```diff
@@ -0,0 +1,13 @@
+welcome to akash file
+
+asd
+fa
+sdf
```

Comment:
hey what did you do here ? its not the right way to do.
change this code

Reply:
this is the correct approach